File: Python/8py 2/lovefunc.py
"""
|--------------------------------------------------------------------------
| Opposites Attract
|--------------------------------------------------------------------------
|
"""

def lovefunc( flower1, flower2 ):
   return (flower1 + flower2) % 2 != 0


lovefunc = lambda a, b: (a ^ b) & 1


def lovefunc( flower1, flower2 ):
   return bool((flower1+flower2)%2)


def lovefunc(a, b):
   return a % 2 ^ b % 2


def lovefunc( flower1, flower2 ):
   return (flower1 + flower2) % 2 == 1


print(lovefunc(1,4), True)
print(lovefunc(2,2), False)
print(lovefunc(0,1), True)
print(lovefunc(0,0), False)

File: PHP/8php 2/lovefunc.php
<?php
/*
|--------------------------------------------------------------------------
| Opposites Attract
|--------------------------------------------------------------------------
|
*/

function lovefunc($flower1, $flower2) {
   return ($flower1 + $flower2) % 2 !== 0;
}


function lovefunc($f1, $f2):bool {
   return ($f1+$f2)%2;
}


function lovefunc($flower1, $flower2) {
   return 1 == ($flower1 + $flower2) % 2; 
}


function lovefunc($f1, $f2) {
   return ($f1 + $f2) % 2 == 1;
}


function lovefunc($flower1, $flower2) {
   return ($flower1 % 2) == !($flower2 % 2);
}


print_r(lovefunc(1, 4). ' ' . true . '
');
print_r(lovefunc(2, 2). ' ' . false . '
');
print_r(lovefunc(0, 1). ' ' . true . '
');
print_r(lovefunc(0, 0). ' ' . false . '
');